core/connector/src/main/java/org/exoplatform/ecm/connector/platform/ManageDocumentService.java

```diff
@@ -686,13 +661,22 @@ private Node getNode(String driveName, String workspaceName, String currentFolde
       return node;
     }
     for (String folder : currentFolder.split("/")) {
-      if (node.hasNode(folder)){
+      if (node.hasNode(folder)) {
         node = node.getNode(folder);
-        if (node.isNodeType(NodetypeConstant.EXO_SYMLINK)) node = linkManager.getTarget(node);
-      } else if (node.isNodeType(NodetypeConstant.EXO_SYMLINK)) {
-        node = linkManager.getTarget(node).getNode(folder);
       } else {
-        return node;
+        // create new folder
+        String name = Text.escapeIllegalJcrChars(org.exoplatform.services.cms.impl.Utils.cleanString(folder));
+        name = (StringUtils.isEmpty(name)) ? DEFAULT_NAME : name;
+        Node newNode = node.addNode(name, NodetypeConstant.NT_FOLDER);
```

Comment:
which session is used to create folder, what are the default permissions added into this folder ? Is it 'any' added by default here or does it inherit default permissions from parent. Does a dedicated permission is added to the current user ? IMO it shouldn't, because the user can revoke its membership in space and consequently, he shouldn't be able to access this folder neither modify its content.

Reply:
checked , the folder inherit default permissions from parent, The permissions set for the admins and the space members